lib/jormungandr/src/Cardano/Wallet/Jormungandr/Api.hs

```diff
@@ -186,11 +184,6 @@ instance MimeRender JormungandrBinary (Tx, [TxWitness]) where
         runPut $ withHeader MsgTypeTransaction $
             putSignedTx mempty ins outs wits
 
-instance MimeRender JormungandrBinary (ByteString, (Tx, [TxWitness])) where
```

Comment:
We would need `(payload, tx, witnesses, auth)` 😩 

I simply removed it here and used a single `ByteString` as return type of `mkCertificateTx`. Maybe we could use the `Fragment` type. Or some new `GenericTx` type.